ocpp-json/src/main/kotlin/com/izivia/ocpp/json/OcppJsonParser.kt

```diff
@@ -38,13 +39,13 @@ abstract class OcppJsonParser(private val mapper: ObjectMapper) {
 
     fun parseAsStringPayloadFromJson(messageStr: String): JsonMessage<String>? =
         ocppMsgRegex.matchEntire(messageStr.replace("\n", ""))?.let {
-            val (msgType, msgId, action, payload) = it.destructured
-            JsonMessage(
-                msgType = JsonMessageType.fromId(msgType.toInt()),
-                msgId = msgId,
-                action = action.takeIf { it.isNotBlank() },
-                payload = payload
-            )
+            val (msgType, msgId, param1, param2, payload) = it.destructured
```

Comment:
je suis pas fan du param1 et param2 au milieu des variables nommées. 
je pense que j'aurais plutôt commencer par récupérer le msgType tout seul (`val msgType = it[0]`) pour ensuite faire le when dessus et déstructurer en variables bien nommées après en fonction du type. Ca fait un peu plus de redondance avec une destructuration très similaire, mais je trouverait ça plus clean 

Qu'en penses tu ?